src/carousel.coffee

```diff
@@ -81,6 +81,11 @@ Ember.Widgets.CarouselComponent = Ember.Component.extend
         $active.removeClass(['active', direction].join(' '))
         @set 'sliding', no
 
+        # Send an action out to signal that the sliding transition has ended
+        # It can is useful to avoid having observers in any parentView having
+        # this component.
+        @send 'transitionEnded'
```

Comment:
So what do you suggest to put in when we add an action to our code without giving any context? 
I'm not sure but I think general concept around observers/actions may be irrelevant to people who knew it, but not necessary to be useless for people who don't.